### PR TITLE
Added FixDaDatabase project

### PR DESCRIPTION
**Pull request by jaapbrasser**
Added the FixDaDatabase PowerShell module to the collection of modules.

### CODE REVIEW

File: Modules/FixDaDatabase.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Name: FixDaDatabase
+
+## Description:
+This module has been created to remediate a common problem with Direct Access servers. It can determine if an Index is present for the Direct Access is present. And it can either remove and create that index.
```
Comment (ClaudioESSilva) [inline Modules/FixDaDatabase.md]: Please review this statement: "determine if an Index is present for the Direct Access is present." double "is present"? 😄

File: Modules/FixDaDatabase.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+| Works with | Author(s) | Project URL(s) | Creation Date | Tags |
+|------------|--------|-------------------|---------------|------|
+| DirectAccess 2012 (R2) | Jaap Brasser ([b](https://www.jaapbrasser.com) \|  [t](https://twitter.com/Jaap_Brasser)) | [GitHub](https://github.com/jaapbrasser/FixDaDatabase) | Apr 2016 | PowerShell, DirectAccess, Database |
```
Comment (ClaudioESSilva) [inline Modules/FixDaDatabase.md]: Right the month full name "April" without abbreviation

File: Modules/FixDaDatabase.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+____
+### Shared blog posts:
+Jaap Brasser ([b](https://www.jaapbrasser.com) \|  [t](https://twitter.com/Jaap_Brasser))
+- [Fix DirectAccess Windows Internal Database – PowerShell Module](https://www.jaapbrasser.com/fix-directaccess-windows-internal-database-powershell-module/)
+____
+### Extended documentation:
+Jaap Brasser ([b](https://www.jaapbrasser.com) \|  [t](https://twitter.com/Jaap_Brasser))
+- [FixDaDatabase - ReadTheDocs.io](http://fixdadatabase.readthedocs.io/en/latest/)
```
Comment (ClaudioESSilva) [inline Modules/FixDaDatabase.md]: Please add this link on the "Project URL(s)" cell up there with 
[ReadTheDocs](http://fixdadatabase.readthedocs.io/en/latest/)

File: README.md [hunk 4]
```diff
@@ -29,6 +29,7 @@ Please, read the [CONTRIBUTING.md](https://github.com/ClaudioESSilva/PowerShellM
 * Modules
   * [dbatools](https://github.com/ClaudioESSilva/PowerShellModulesCentral/blob/master/Modules/dbatools.md)
   * [DRSRule](https://github.com/ClaudioESSilva/PowerShellModulesCentral/blob/master/Modules/DRSRule.md)
+  * [FixDaDatabase](https://github.com/ClaudioESSilva/PowerShellModulesCentral/blob/master/Modules/FixDaDatabase.md)
```
Comment (ClaudioESSilva) [inline README.md]: Please, remove readme.md from the PR.
I will add it later.

**Review by ClaudioESSilva — CHANGES_REQUESTED** [review]
Please review my comments.

**Comment by ClaudioESSilva** [comment]
Thanks for your PR @jaapbrasser !
Please, view my comments on the changes request.

**Comment by jaapbrasser** [comment]
I did my best, let me know if it is up to spec.

**Review by ClaudioESSilva — APPROVED** [review]
Good to go!

**Comment by ClaudioESSilva** [comment]
Merged!
Thanks again for your contribution! Looking forward to get more PRs from you! 😄

**Pull request merged**